bin/jb_report

```
@@ -0,0 +1,84 @@
+#
+# [DISTRIBUTION STATEMENT A] This material has been approved for public release and unlimited distribution.  Please see
+# Copyright notice for non-US Government use and distribution.
+#
+# This Software includes and/or makes use of Third-Party Software subject to its own license.
+#
+# DM21-0884
+#
+# ======================================================================================================================
+
+import argparse
+import logging
+
+import juneberry.reporting.utils as jb_report_utils
+import juneberry.scripting as jbscripting
+
+logger = logging.getLogger("juneberry.jb_report")
+
+
+def setup_args(parser) -> None:
+    """
+    Adds arguments to the parser
+    :param parser: The parser in which to add arguments.
+    """
+
+    parser.add_argument('-e', '--experiment', help='Name of the experiment directory whose experiment config contains '
+                                                   'a reports stanza indicating which reports to build.')
+    parser.add_argument('-m', '--model', help='Name of the model directory whose model config contains a reports '
+                                              'stanza indicating which reports to build.')
+    parser.add_argument('-f', '--file', action='append', help='Name of one or more JSON files containing a reports '
+                                                              'stanza. These files do not need to be affiliated with a '
+                                                              'particular experiment or model.')
+
+
+def main():
+    parser = argparse.ArgumentParser(description='This script generates report files for models and experiments.')
+    setup_args(parser)
+    jbscripting.setup_args(parser)
+    args = parser.parse_args()
+
+    # Set up logging.
+    jbscripting.setup_logging_for_script(args)
+
+    report_list = None
+
+    # Fetch the desired reports for the experiment and build each report.
+    if args.experiment:
```

What if multiple different options are selected?  It looks like -f wins and the others are silently ignored.